src/languages/hu/commands/replay.json

```
@@ -0,0 +1,10 @@
+{
+	"nosong": {
+		"title": "Nem játszom",
+		"description": "Nem játszok semmit"
+	},
+	"success": {
+		"title": "dal újrajátszása"",
```

```suggestion
		"title": "dal újrajátszása",
```
You added one " more than needed